### PR TITLE
[MRG] Ident info adapted for Germany

### PR DESCRIPTION
**Pull request by Kirastina**

### CODE REVIEW

File: site/content/french/increase-legal-weight.md [hunk 1]
```diff
@@ -92,64 +92,64 @@ plus sa force probante est élevée.
...
-**Force probante maximum** 
-## Signature électronique qualifiée (SEQ)
-Déverrouillez la norme de signature électronique 
-la plus élevée (SEQ) en vous identifiant en ligne par appel vidéo 
-ou par contact personnel avec l’ID-app de Swisscom.
+**Force probante élevée** 
+# Signature électronique avancée (SEA)
+Vous activez la SEA en connectant votre compte Skribble avec un 
+numéro de téléphone portable suisse. Vous pouvez le paramétrer 
+dans votre profil. Si vous êtes un client Skribble Business ou Enterprise, vous pouvez
```
Comment (Kirastina) [inline site/content/french/increase-legal-weight.md:128]:
```suggestion
dans votre profil. Si vous êtes un client Skribble Business ou Enterprise, vous pouvez demander l'activation de la SEA via l'adresse électronique de l'entreprise (SEA pour Business).
```

File: site/content/french/increase-legal-weight.md [hunk 2]
```diff
@@ -92,64 +92,64 @@ plus sa force probante est élevée.
...
-## Signature électronique qualifiée (SEQ)
-Déverrouillez la norme de signature électronique 
-la plus élevée (SEQ) en vous identifiant en ligne par appel vidéo 
-ou par contact personnel avec l’ID-app de Swisscom.
+**Force probante élevée** 
+# Signature électronique avancée (SEA)
+Vous activez la SEA en connectant votre compte Skribble avec un 
+numéro de téléphone portable suisse. Vous pouvez le paramétrer 
+dans votre profil. Si vous êtes un client Skribble Business ou Enterprise, vous pouvez 
+demander l'activation de la SEA via l'adresse électronique
```
Comment (Kirastina) [inline site/content/french/increase-legal-weight.md:129]:
```suggestion
```

File: site/content/french/increase-legal-weight.md [hunk 3]
```diff
@@ -92,64 +92,64 @@ plus sa force probante est élevée.
...
-Déverrouillez la norme de signature électronique 
-la plus élevée (SEQ) en vous identifiant en ligne par appel vidéo 
-ou par contact personnel avec l’ID-app de Swisscom.
+**Force probante élevée** 
+# Signature électronique avancée (SEA)
+Vous activez la SEA en connectant votre compte Skribble avec un 
+numéro de téléphone portable suisse. Vous pouvez le paramétrer 
+dans votre profil. Si vous êtes un client Skribble Business ou Enterprise, vous pouvez 
+demander l'activation de la SEA via l'adresse électronique 
+de l'entreprise (SEA pour Business).
```
Comment (Kirastina) [inline site/content/french/increase-legal-weight.md:130]:
```suggestion
```

File: site/content/german/identification.md [hunk 4]
```diff
@@ -123,31 +123,23 @@ Wann nutze ich QES, wann FES?
...
 {{% markdown %}}
 ## Verträge mit Formvorschrift
-Für Verträge mit Formvorschrift oder hohem Abspruch an Beweiskraft wird typischerweise die qualifizierte elektronische Signatur (QES) eingesetzt. Bei der QES setzt das Gesetz voraus, dass die Identität der Signierenden im Voraus im persönlichen Kontakt geprüft wird.
+Für Verträge mit Formvorschrift oder hohem Abspruch an Beweiskraft wird typischerweise die qualifizierte elektronische Signatur (QES) eingesetzt. Die Identifikationsanforderungen bei der QES sind hoch, die Prüfung muss persönlich oder online via Video stattfinden.
 {{% /markdown %}}
 {{< /content >}}
 
 {{< picture contract-with-requirement 480 "Eine Frau mit einem Handy blickt hinter einem Vertrag hervor und davor steht ein Geschäftsmann mit einem Laptop." >}}
 
 {{< content text=center top=8 bottom=0 width=narrow >}}
```
Comment (Kirastina) [inline site/content/german/identification.md:132]:
```suggestion
{{< content text=center top=12 bottom=8 width=narrow >}}
```

File: site/content/german/identification.md [hunk 5]
```diff
@@ -157,27 +149,22 @@ Die QES schalten Sie frei, in dem Sie sich online via Video-Call oder im persön
 {{< picture id-easy 270 "Eine Frau und ein Mann stehen vor einem grossen Softwarefenster und der Mann haltet eine grosse Luppe mit einer Identitätskarte drin." >}}
 
 
-Ihr führt die Identifikationen selbst durch.
+Ein Vertreter Ihres Unternehmens durchläuft eine einmalige Schulung. Anschliessend kann er die Signierenden selbst im persönlichen Kontakt identifizieren. Passend bei einer grossen Anzahl QES-Signierender.
```
Comment (Kirastina) [inline site/content/german/identification.md:152]:
```suggestion
Ein Vertreter Ihres Unternehmens durchläuft eine einmalige Schulung. Anschliessend kann er die Signierenden selbst im persönlichen Kontakt identifizieren. Passend für Unternehmen mit einer grossen Anzahl QES-Signierender.
```

File: site/content/english/identification.md [hunk 6]
```diff
@@ -23,26 +23,24 @@ The procedures differ depending on the application.
...
+The simple electronic signature (SES) or the advanced electronic signature (AES) is typically used for contracts without formal requirements. The identification requirements for EES and FES are low to medium.
 {{% /markdown %}}
 {{< /content >}}
 
 {{< picture contract-no-requirement 480 "A woman sits on a contract, a man looks out from behind the contract." >}}
 
 {{< content text=center top=16 bottom=8 width=narrow >}}
 {{% markdown %}}
-**Identification options for the advanced electronic signature (AES):**
+**Identification options for contracts without written form:
```
Comment (Kirastina) [inline site/content/english/identification.md:34]:
```suggestion
**Identification options for contracts without written form:**
```

File: site/content/english/identification.md [hunk 7]
```diff
@@ -181,9 +167,7 @@ Appropriate if new QES signers are constantly being added.
 
 {{< picture id-direct 210 "A man is sitting in front of a laptop, above him is a sotware window, where a woman stands out with a big magnifying glass and checks an identity card." >}}
 
-Enable the highest e-signature standard QES from home.
-
-Suitable for individual signatories.
+The signatories enable the QES by having themselves identified online via video (EU) or face-to-face at an identification site (Switzerland). Suitable for individuals and companies with a small number of QES signatories.
 &nbsp;
```
Comment (Kirastina) [inline site/content/english/identification.md:171]:
```suggestion

```

Comment (Kirastina) [inline site/content/english/identification.md:171]:
```suggestion

```

**Review by Kirastina — COMMENTED** [review]
Korrekturen Identification EN

File: site/content/french/identification.md [hunk 8]
```diff
@@ -24,24 +24,24 @@ Les procédures diffèrent en fonction de la demande.
...
 {{< /content >}}
 
 {{< content text=left width=wide display=flex >}}
 
 {{< plan
   white
-  ""
-  "Adresse électronique privée"
+  "SES"
+  "Adresse électronique"
```
Comment (Kirastina) [inline site/content/french/identification.md:44]:
```suggestion
  "Adresse électronique
  &nbsp;"
```

File: site/content/french/identification.md [hunk 9]
```diff
@@ -41,7 +41,8 @@ Pour les contrats sans exigences formelles, la signature électronique simple (S
 {{< plan
   white
   "SES"
-  "Adresse électronique"
+  "Adresse électronique
+  &nbsp;"
```
Comment (Kirastina) [inline site/content/french/identification.md:45]:
```suggestion
```

File: site/content/french/identification.md [hunk 10]
```diff
@@ -41,7 +41,8 @@ Pour les contrats sans exigences formelles, la signature électronique simple (S
 {{< plan
   white
   "SES"
-  "Adresse électronique"
+  "Adresse électronique
```
Comment (Kirastina) [inline site/content/french/identification.md:44]:
```suggestion
  "Adresse électronique"
```

File: site/content/french/identification.md [hunk 11]
```diff
@@ -24,24 +24,24 @@ Les procédures diffèrent en fonction de la demande.
...
+Pour les contrats sans exigences formelles, la signature électronique simple (SES) ou la signature électronique avancée (SEA) est généralement utilisée. Les exigences d'identification pour la SES et la SEA sont faibles à moyennes.
 {{% /markdown %}}
 {{< /content >}}
 
 {{< picture contract-no-requirement 480 "Une femme est assise sur un contrat, un homme regarde derrière le contrat." >}}
 
 {{< content text=center top=16 bottom=8 width=narrow >}}
 {{% markdown %}}
-**Options d'identification pour la signature électronique avancée (SEA):**
+**Options d'identification pour contrats sans exigence formelle:**
```
Comment (Kirastina) [inline site/content/french/identification.md:35]:
```suggestion
**Options d'identification pour les contrats sans exigence formelle:**
```

File: site/content/french/identification.md [hunk 12]
```diff
@@ -126,35 +126,24 @@ Quand dois-je utiliser la SEQ, quand la SEA ?
...
-
-{{< content text=center top=8 bottom=4 width=narrow >}}
+{{< content text=center top=8 bottom=8 width=narrow >}}
 {{% markdown %}}
-**Service d'identification**
-Déverrouillez la norme de signature électronique la plus élevée (SEQ) 
-en vous identifiant en ligne par appel vidéo ou par contact personnel 
-avec l’ID-app de Swisscom.
-&nbsp;
+**Options d'identification pour contrats avec exigences formelles:**
```
Comment (Kirastina) [inline site/content/french/identification.md:137]:
```suggestion
**Options d'identification pour les contrats avec exigences formelles:**
```

File: site/content/french/identification.md [hunk 13]
```diff
@@ -41,7 +41,7 @@ Pour les contrats sans exigences formelles, la signature électronique simple (S
 {{< plan
   white
   "SES"
-  "Adresse électronique"
+  "Adresse électronique privé"
```
Comment (Kirastina) [inline site/content/french/identification.md:44]:
```suggestion
  "Adresse électronique privée"
```

File: site/content/english/enable-qes.md [hunk 14]
```diff
@@ -14,8 +14,7 @@ Maximise the legal weight
 of your electronic signature
 ===============	
 Enable the highest e-signing standard (QES) by getting 
-yourself identified online via video-call or by personal contact 
-with the ID app by Swisscom.
+yourself identified online via video (EU) or by personal contact (Switzerland).
```
Comment (Kirastina) [inline site/content/english/enable-qes.md:17]:
```suggestion
yourself identified online via video (EU) or face-to-face (Switzerland).
```

File: site/content/english/enable-qes.md [hunk 15]
```diff
@@ -43,11 +42,11 @@ with our partners Swisscom Trust Services and IdentityTM.
 
 {{< content text=center top=16 width=narrow >}}
 {{% markdown %}}
-Identification 
+Switzerland: Identification
```
Comment (Kirastina) [inline site/content/english/enable-qes.md:45]:
```suggestion
Switzerland: Face-to-face
```

File: site/content/english/enable-qes.md [hunk 16]
```diff
@@ -43,11 +42,11 @@ with our partners Swisscom Trust Services and IdentityTM.
 
 {{< content text=center top=16 width=narrow >}}
 {{% markdown %}}
-Identification 
+Switzerland: Identification 
 by personal contact
```
Comment (Kirastina) [inline site/content/english/enable-qes.md:46]:
```suggestion
identification
```

**Review by Kirastina — COMMENTED** [review]
Replace "in personal contact" with "face-to-face"

File: site/content/german/enable-qes.md [hunk 17]
```diff
@@ -42,11 +42,11 @@ Die Identifikation wird in Zusammenarbeit mit unseren Partnern Swisscom Trust Se
 
 {{< content text=center top=16 width=narrow >}}
 {{% markdown %}}
-Identifikation 
+Schweiz: Identifikation 
 im persönlichen Kontakt
 ===============
-Die QES schalten Sie frei, in dem Sie sich im persönlichen Kontakt
-mit der ID-App von Swisscom identifizieren lassen.
+Lassen Sie sich im persönlichen Kontakt
```
Comment (Kirastina) [inline site/content/german/enable-qes.md:48]:
```suggestion
Kommen Sie an einem unserer Identifikationsstandorte vorbei.
```

File: site/content/german/enable-qes.md [hunk 18]
```diff
@@ -42,11 +42,11 @@ Die Identifikation wird in Zusammenarbeit mit unseren Partnern Swisscom Trust Se
...
+mit der ID-App von Swisscom identifizieren.
```
Comment (Kirastina) [inline site/content/german/enable-qes.md:49]:
```suggestion
Wir identifizieren Sie kostenlos innert weniger Minuten.
```

**Review by Kirastina — COMMENTED** [review]
Untertitel dupliziert nicht mehr Titel.

File: site/content/french/enable-qes.md [hunk 19]
```diff
@@ -41,11 +41,12 @@ L'identification est proposée en coopération avec nos partenaires Swisscom Tru
...
 {{< content text=center top=16 width=narrow >}}
 {{% markdown %}}
-Identification 
+Suisse: Identification 
 par contact personnel
 ===============
-Déverrouillez la norme de signature électronique la plus élevée (SEQ) 
-en vous identifiant par contact personnel avec l’ID-app de Swisscom.
+
+Laissez-vous identifier par contact personnel
```
Comment (Kirastina) [inline site/content/french/enable-qes.md:48]:
```suggestion
Passez à l'un de nos lieux d'identification.
```

File: site/content/french/enable-qes.md [hunk 20]
```diff
@@ -41,11 +41,12 @@ L'identification est proposée en coopération avec nos partenaires Swisscom Tru
...
+avec l’ID-app de Swisscom.
```
Comment (Kirastina) [inline site/content/french/enable-qes.md:49]:
```suggestion
Nous vous identifierons gratuitement en quelques minutes.
```

**Review by Kirastina — COMMENTED** [review]
Untertitel wiederholt nicht mehr Titel

File: site/content/english/enable-qes.md [hunk 21]
```diff
@@ -43,11 +42,11 @@ with our partners Swisscom Trust Services and IdentityTM.
...
 ===============
-Enable the highest e-signing standard (QES) by getting yourself identified
-by personal contact with the Swisscom ID app.
+Get yourself identified by personal contact
```
Comment (Kirastina) [inline site/content/english/enable-qes.md:48]:
```suggestion
Drop in at one of our identification sites.
```

File: site/content/english/enable-qes.md [hunk 22]
```diff
@@ -43,11 +42,11 @@ with our partners Swisscom Trust Services and IdentityTM.
...
+with the Swisscom ID app.
```
Comment (Kirastina) [inline site/content/english/enable-qes.md:49]:
```suggestion
We will identify you free of charge within a few minutes.
```

**Review by Kirastina — COMMENTED** [review]
Untertitel wiederholt nicht mehr Titel

**Comment by TillLord** [comment]
I checke the meta title and meta description: Looks good.

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested it and it looked ok. I found a bug, regarding a text on the identifications page and contacted @Kirastina about it. Will be fixed in the next days.

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested the mobiliar change again and it looks good on my mobile phone.

**Comment by Kirastina** [comment]
(a few content updates will follow in a second merge)

**Pull request merged**